massa-async-pool/src/changes.rs

```diff
@@ -0,0 +1,54 @@
+//! Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+//! This file provides structures representing changes to the async message pool
+
+use crate::message::{AsyncMessage, AsyncMessageId};
+
+/// Enum representing a add/delete change on a value T
+#[derive(Debug, Clone)]
+pub enum AddOrDelete<T> {
+    /// adds a new value T
+    Add(T),
+
+    /// deletes the value T
+    Delete,
+}
+
+/// allows applying another AddOrDelete to the current one
+impl<T> AddOrDelete<T> {
+    fn apply(&mut self, other: Self) {
```

Comment:
```suggestion
impl<T> AddOrDelete<T> {
   /// allows applying another AddOrDelete to the current one
    fn apply(&mut self, other: Self) {
```

Comment in the wrong place